man/systemd.network.xml

```diff
@@ -1430,6 +1430,18 @@
             sent even if this is set to true.</para>
           </listitem>
         </varlistentry>
+
+        <varlistentry>
+          <term><varname>SendMUDURL=</varname></term>
```

Comment:
This should be MUDURL=, not Send*=. In other cases, we SendFoo= is used for a boolean whether to send Foo= value. (The only exception is SendOption=, but Option= would be confusing as an option.)